### DISCUSSION TITLE
Question about README.md content

### DISCUSSION THREAD

**Discussion by nextdoorpanda** [sheabunge/code-snippets#112] [General]
Hi!
I'm translating Code Snippets to Greek and I cannot understand what "another side" means in README.md's sentence "Snippets can be exported for transfer to another side.....". 
Could you please shed some light on that?

**Comment by sheabunge** [comment]
Hi @nextdoorpanda,

It looks like you've discovered a typographical error. The text should read: "Snippets can be exported for transfer to another site".

Thank you for your work translating!

**Reply by nextdoorpanda** [reply]
Oh, ok! I'll fix it then! Thank you for your reply! :)
